code/game/objects/items/weapons/implants/implantcase.dm

```diff
@@ -121,3 +121,15 @@
 		src.imp = new /obj/item/weapon/implant/death_alarm( src )
 		..()
 		return
+
+//Pax(Peace) Implant Case
+/obj/item/weapon/implantcase/peace
+	name = "Glass Case- 'Pax'"
```

Comment:
The capitalization here is incorrect, it should read `glass case - 'Pax'`, otherwise it gets treated as a proper noun.